docker/utils/startup.sh

```diff
@@ -1,17 +1,14 @@
 #!/bin/bash -xeu
 
-# legacy support
-# This environment variable was used to set the gatewway cluster host in standalone and embedded mode.
-# Now, there are two dedicated environment variables for the two different deployment scenarios.
-export ZEEBE_HOST=${ZEEBE_HOST:-$(hostname -i)}
-# Legacy support
+HOST=$(hostname -i)
 
 if [ "$ZEEBE_STANDALONE_GATEWAY" = "true" ]; then
-    export ZEEBE_GATEWAY_CLUSTER_HOST=${ZEEBE_GATEWAY_CLUSTER_HOST:-${ZEEBE_HOST}}
+    export ZEEBE_GATEWAY_NETWORK_HOST=${ZEEBE_GATEWAY_NETWORK_HOST:-${HOST}}
+    export ZEEBE_GATEWAY_CLUSTER_HOST=${ZEEBE_GATEWAY_CLUSTER_HOST:-${HOST}}
```

Comment:
Should we align this with line https://github.com/camunda-cloud/zeebe/pull/6924/files#diff-4fcb53c4d5d3bf431c6b9328948794f68f68fe0f74f4fbc75dd6b4b489ec48ddR12, i.e. if a `ZEEBE_GATEWAY_NETWORK_HOST` is set but `ZEEBE_GATEWAY_CLUSTER_HOST` is not set then `ZEEBE_GATEWAY_NETWORK_HOST` is used instead of `HOST`? Or is there a reason why it differs?

```suggestion
    export ZEEBE_GATEWAY_CLUSTER_HOST=${ZEEBE_GATEWAY_CLUSTER_HOST:-${ZEEBE_GATEWAY_NETWORK_HOST}}
```

Reply:
Good point :+1: